migrations/insertSampleData.js

```diff
@@ -11,37 +11,37 @@ Models.users.insertMany(users, (err, data) => {
   if (err) {
     console.log('Error:', err);
   }
-  console.log(`Inserted ${data.insertedCount} users.`);
+  // console.log(`Inserted ${data.insertedCount} users.`);
```

Comment:
I suggest overriding the rule in this file again - in the data seeding process it's useful to get those log statements triggered.